client/src/components/NavButton.js

```diff
@@ -10,9 +11,16 @@ const NavButton = styled.button`
   border: none;
   outline: none;
 
-  :hover {
-    outline: ${props => props.theme.colors.quatenary}, opacity:1;
+  &:hover {
+    color: ${props => props.theme.colors.quartenary};
   }
+  &:active {
+  }
+  color: ${props => props.theme.colors.quartenary};
 `;
 
+function NavButton(props) {
```

Comment:
You don't need this additional component. Export your styled component here.
1: remove NavButton
2: rename ThisNavButton to NavButton

Reply:
Works- thank you.

Reply:
Thank you...